### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by mallorym**
This project was a little out of date. Updated dependencies and added a README.

I also changed the feature up a bit to point to Modus Labs. Because the newer dependencies changed the structure of the tests a bit, I had to update the step_def and config file. Test passed locally.

### CODE REVIEW

**Review by a-arias — APPROVED** [review]
looks good 👍

File: nightwatch.conf.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+const seleniumServer = require('selenium-server');
+const phantomjs = require('phantomjs-prebuilt');
```
Comment (izaac) [inline nightwatch.conf.js]: should we still use the deprecated `phantomjs`? I don't even see it in the Nightwatch documentation: http://nightwatchjs.org/gettingstarted#browser-drivers-setup maybe that now we have Headless and Firefox headless modes? 👍

**Review by izaac — APPROVED** [review]

File: nightwatch.conf.js (hunk 1 above)
Comment (mallorym) [reply nightwatch.conf.js]: Great point! I'll look at that.

**Comment by mallorym** [comment]
Removed phantomjs dependency. This is good to merge :shipit:

**Pull request merged**